supporting_content/metrics/billedunitprice.md

```diff
@@ -0,0 +1,67 @@
+# Column: BilledUnitPrice
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | CUR;                         | Use lineItem/NetUnblendedRate if available, otherwise ​​lineItem/UnblendedRate. |
+| GCP       | BigQuery Billing Export;<br>BigQuery Pricing Data Export;<br>Cloud Billing API | Not available in BigQuery Billing Export<br>**price.effective_price** (available only in detail export) is the closest match for a subset of resources/services; doesn't include all relevant discounts/credits.<br>*see Discussion / Scratch space - GCP column mappings related issues for more details* |
+| Microsoft | Cost details;<br>*TODO: Add reference to Custom Azure Prices REST API* | **unitPrice** is probably the best match, but according to  documentation it's available only for EA and PAYG. <br>*see Discussion / Scratch space - Microsoft column mappings related issues for more details* |
+| OCI       | Cost and Usage Report        | **cost/unitPrice** is the best match<br>*see Discussion / Scratch space - OCI column mappings related issues for more details* |
```

Comment:
In these cases, are we sure these are the unit prices inclusive of all the discounts (mainly including commitment discounts?)

Reply:
Inclusive of discounts but not amortized